From 5a4f441d05858ec217a5c64c4707c3a7d4674c7d Mon Sep 17 00:00:00 2001
From: Marc Titinger <mtitinger@baylibre.com>
Date: Fri, 29 Jan 2016 16:35:35 +0100
Subject: [PATCH] rsync the floder passed as arg2 of stop hook to powerci

Signed-off-by: Marc Titinger <mtitinger@baylibre.com>
---
 iio-probe-stop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/iio-probe-stop b/iio-probe-stop
index 9dfb99f..838a3d6 100755
--- a/iio-probe-stop
+++ b/iio-probe-stop
@@ -22,3 +22,6 @@ if [ -f /tmp/gnuplot-$1 ]
 then
 	gnuplot /tmp/gnuplot-$1
 fi
+
+
+rsync -r $2 powerci@powerci:/var/www/html/kernel-ci/attachments
-- 
2.7.0.rc3

